### DISCUSSION TITLE
Embroidery machines discussion

### DISCUSSION THREAD

**Discussion by lexelby** [inkstitch/inkstitch#2422] [General]
I just noticed this brand new machine on the market: [EverSewn Sparrow X](https://www.amazon.com/EverSewn-Sparrow-Next-Generation-Embroidery-Machine-Customize/dp/B07JYGPM1Q).  What do you folks think?

As far as I can tell, at $800, this is now by far the cheapest machine that can trim jump stitches.  The next cheapest one, I believe, is the Janome Memory Craft 400E at $1300.

**Comment by ndrwclrk** [comment]
Looks pretty sweet.  Seems as though EverSewn has [some affiliation with Bernina](http://www.sewingreport.com/2017/01/do-you-need-an-expensive-sewing-machine/)

**Comment by jameskolme** [comment]
I think there is some continental price manipulation going on, currently I can buy brand new Janome Memory Craft 500 around 900e, that is cheapest I can find.

**Comment by farfadet46** [comment]
Hi, i work in a fablab in southern France and we have the inov'is 955 a realy cool sewig / embroidery machine :) bu it's near 1000€

in search we can found the BROTHER Innovis M240ED is at 700€ some promotionnal give it at low price like his one https://www.coudreetbroder.com/machine-a-broder-brother-innovis-m240ed-garantie-5-ans,fr,4,BROTHERINNOVISM240ED.cfm (from France)

**Comment by ndrwclrk** [comment]
Innovis 955 = SE400, which is priced much lower in the USA (~300USD).
I don't see a comparable model to the Innovis M240ED on brother's USA or Canada website. It's a constant source of aggravation that sewing machine prices are so much lower in the USA, even than in Canada (even when taking exchange rates into account).  Lots of Canadians end up cross-border shopping, and then taking the machines home with a voided warranty. I did that very thing, but the machines are so reliable that it's not a big deal.  I've also taken mine in for preventive maintenance at a local dealer, and encountered no problems.

**Comment by lexelby** [comment]
Nope, that Sparrow X is a total dud. :(

I bought one and was super excited.  I knew there was a bit of a risk since it's such a new product and there are only 2 amazon reviews, but I decided to take a chance.  Unfortunately, it didn't work out.

The thread trimmer totally doesn't work.  I mean, it trims the thread, but at least half the time it loses the thread immediately when it starts stitching again and I have to rethread it.  What's the point of that?

The thread trimmer is pretty much the entire reason I bought this thing, so I'm returning it.

**Comment by X3msnake** [comment]
thread losing after cut is usually a sign of too much tension

No dia domingo, 30 de dezembro de 2018, Lex Neva <notifications@github.com>
escreveu:

> Nope, that Sparrow X is a total dud. :(
>
> I bought one and was super excited. I knew there was a bit of a risk since
> it's such a new product and there are only 2 amazon reviews, but I decided
> to take a chance. Unfortunately, it didn't work out.
>
> The thread trimmer totally doesn't work. I mean, it trims the thread, but
> at least half the time it loses the thread immediately when it starts
> stitching again and I have to rethread it. What's the point of that?
>
> The thread trimmer is pretty much the entire reason I bought this thing,
> so I'm returning it.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/inkstitch/inkstitch/issues/361#issuecomment-450538067>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKke-hW3Xq65zYoFg7ZaWw344x3qTRl1ks5u-D1sgaJpZM4Y4xBY>
> .
>


-- 
Com os melhores cumprimentos,
Vinicius Silva

**Comment by lexelby** [comment]
Yup, I suspected as much. I kept lowering the tension until it created a bird's nest under the fabric and jammed the machine, but it didn't help.

**Comment by X3msnake** [comment]
both tensions must be lowered. i did a tensionmeter with a pen and a spring to tension the bottom and upper thread. 

but you can do it visually, just start by lowering the bottom tension

i used inkscape spiral tool and inkstitched a 10mm satin to visually find the upper tension

https://github.com/FabricaModerna/Embroidery-Machine-Happy/issues/3

**Comment by lexelby** [comment]
This machine doesn't even have the option to change the lower tension.

**Comment by X3msnake** [comment]
really? not with s screw or something. that sucks :(

**Comment by lexelby** [comment]
Maybe way down deep inside, but there's no mention of it in the manual, and I'm not gonna tinker with it for fear of them refusing to accept a return.

**Comment by wwderw** [comment]
I would always start with changing the upper tension 1st.

Changing the bottom tension is more of an issue of getting it back to where it's kosher for a lot of projects.

Do you have an "i-test" pattern to make sure that tension is good or not?  Depending on that test, it should tell you if it needs to be tightened, loosened or look for another culprit.

**Comment by lexelby** [comment]
Oh, I definitely got the tension right.  It started off way too tight, with the telltale bobbin thread showing on the top.  I lowered it in successive tests until I got a good result, but it still lost the thread after trimming.  I lowered it yet further and made a terrible bird's nest underneath that was quite difficult to clean out.

I tried a lot of other things too, with no luck.  Bottom line is, there's a reason this is the cheapest machine to have jump stitch trimming: it's poorly designed.  I wrote all my tests up in an Amazon review which should hopefully post soon, if you're interested.

**Comment by markol** [comment]
Workaround for this thread loosing might be to instruct the carriage to move some distance (20 mm?) to pull out some thread excess, and then perform the cut.  I do believe it should be easily doable in pattern generation stage, later on if this works, you can call the support to release firmware update.

**Comment by lexelby** [comment]
Thanks for the idea, but the machine already does that.  It moves a good two inches or so after cutting before returning to the next position and continuing.

I appreciate everyone's trying to help, but I've already returned the sparrow x.  

If anyone knows of a (big) bargain on a used commercial machine, let me know! :)

**Comment by lexelby** [comment]
Oh, I misunderstood: you meant do a jump _before_ trimming.  Based on how this machine trims, I believe that would only result in leaving a thread tail on the top at the trim location but still leave the same thread tail length hanging from the needle.

**Comment by markol** [comment]
Yes you are right, moment after posting I have realized that is not going to work without additional hacks, for bottom located trimmers.

**Comment by X3msnake** [comment]
@lexelby you should do a quick video review so others know the flaws of that machine before returning it, if you choose not to keep it ;)

**Comment by lexelby** [comment]
Too late, like I said, I already returned it.  Once I was pretty sure it wasn't going to do the job, I wanted to get it out before I accidentally damaged it or something!

**Comment by X3msnake** [comment]
Oh! ok :)

Lex Neva <notifications@github.com> escreveu no dia quarta, 2/01/2019 à(s)
23:23:

> Too late, like I said, I already returned it. Once I was pretty sure it
> wasn't going to do the job, I wanted to get it out before I accidentally
> damaged it or something!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/inkstitch/inkstitch/issues/361#issuecomment-451016330>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKke-qaBrq0DGJVALVwrMLXBzB-ztbjDks5u_T9tgaJpZM4Y4xBY>
> .
>


-- 
Com os melhores cumprimentos,
Vinicius Silva

**Comment by kaalleen** [comment]
I'm sorry to hear that this did not work out for you.

Is there any news about this story? #230

**Comment by lexelby** [comment]
None yet.  These things take time, I assume :)

**Comment by jameskolme** [comment]
Hope everything goes well. I was curious about this brand and so I did some extensive search, but very little actual info, even with sewing machines included and using Sherlock hat.

**Comment by lexelby** [comment]
I got a great deal on a Janome 400e.  This thing is super fast and doesn't lose the thread while trimming!  It does seem to sometimes leave a bit of the thread end on top of the fabric, but I'm tinkering with things to improve that.

It only has a usb A port, like a port on a computer, so you're supposed to move a USB drive back and forth.  So boring.

I found an old android phone in my drawer and hacked it to make it pretend to be a USB drive.  Looks like only older android phones can still do this.  I added SambaDroid to make the phone share its sd card on the network, and I added a script to "unplug" and "replug" the phone from the 400e whenever a file changes.  Now my sewing machine is network-enabled :D

**Comment by wwderw** [comment]
Some home machines have a wireless adaptor to plug into the USB ports to transmit via wireless.

Brother switched it to some funky dongle type of deal now though.  They used to have a better version a few yrs back, more like a wireless hub.  But I haven't looked into what all they are doing now, not since the last time I bought a PR, back in 2010.

**Comment by lexelby** [comment]
I guess I effectively made one of those myself out of spare parts :)

**Comment by jameskolme** [comment]
Same excess thread happens to my 500e machine as well, sometimes. Probably something that I do wrong. 500e is possible to connect directly to computer and memory stick, but no wireless connections :P

**Comment by lexelby** [comment]
From what I can tell, the excess thread is due to a fabric that is thick enough or tightly woven enough to "grab" the thread and pull out some extra after a trim.  Setting the thread tail length to 1 definitely helps some.  I also loop the thread around the pre-tensioner one extra time as they show in the instruction manual, and that helps too.

**Comment by tyrosinase** [comment]
Potentially dumb question but: what are the differences between quilting and embroidery machines, beyond the obvious (quilting machine can roll up the work inside the throat of the machine to get more range)? I'm already looking ahead to something larger-format than the 5x7 machine, and wondering how fast I'd outgrow an MC 500e, and a quilting machine came to mind. There's a FLQS just a few train stops away that rents time on their machines, if they're things that accept something Inkstitch can output.

(And also if they let me feed it as unconventional of stuff as I do to my embroidery machine, which is a whole different question: https://silverseams.com/2019/05/27/so-many-dragons/ )

**Comment by lexelby** [comment]
I've been wondering this myself!  My mom has a long-arm quilting machine, but it's a manual affair.  I wonder if there are automated versions that can read DST files?

I can tell you that the 400e's size is pretty darn awesome, and of course the 500e is even bigger.  It would at least take awhile before you outgrew it. :)

@wwderw Do you know anything about automated quilting machines?  Is that a thing?

**Comment by wwderw** [comment]
Big deal.  We have a statler stitcher by gammil.

They have their own format for patterns.  qli for the Statler.  All run stitch, line work.

It would be nice if qli format could be supported, but I just don't know how feasible that would be.

**Comment by tyrosinase** [comment]
Oh, you'd be surprised at how ambitious I can get with this wacky thing. (Current project is a five-hooping project composed of two 10x12 pieces of fabric, two 5x7s, and one 3x3. There is much careful alignment in between hoopings.)

I fully expect I'd still do occasional composite projects in a 500e. :stuck_out_tongue_winking_eye: 

It looks like, from what I see as far as downloadable patterns, most of what the computer-controlled ones do is small repeating patterns... basically just embroidery machine style, except that instead of hooping the fabric you have it on rollers and you scroll it to each position in succession. I'm not sure if that would have sufficient inter-design precision to satisfy me. Should probably visit the FLQS and see how theirs work.

Guess I should just get the magnetic hoop working on the PE-800 (third-party one and the powdercoat is thick enough the pins don't fit in the slots so it needs ground out) and practice my manual alignment.
